active/website.md

```diff
@@ -0,0 +1,63 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
+- Help to make the website accessible to all
+
+
+### Delegated responsibilities:
+- Chair, Co-chair and Board Liaison can make final decision of  sections of the website that might not conform with DSF's CoC.
+- Members can work on bugs
+- Members can tag,label and respond to issues.
+- Member can maintain and monitor the website including updating versions
+- Mentor new contributors to the website
+- Chair, Co-Chair and Board Liaison can sign off on new features
+
+## Initial membership
+
+- Chair: TBA
+- Co-Chair: TBA
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBA
+- Other members:
+    - Sarah Abderemane
+    - Eric Sherman
+    - Mark Walker
+    - Jason Judkins
+    - Paolo Melchiorre
+    - Sanyam Khurana
+    - Tobias McNulty
+    - Ron Maravanyika
+
+
+
+## Future membership
+
+### Who is eligible to join? Any volunteer, or are there specific requirements?
+
+Members must have interest in Django and should be able to work with Django. Most importantly members must be willing to adhere to Django's [Code of Conduct](https://www.djangoproject.com/conduct/) . Members must be well versed with the process of contributing to **djangoproject.com** or at least willing to be guide. We welcome all experience levels, we also welcome first time contributors. 
+
+### How do people who want to join sign up / volunteer / express interest?
+Individuals can express interest by emailing to the working group mailing list at `website-wg@djangoproject.com` 
+
+### How will decisions on adding/removing members be handled?
+Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.
```

Comment:
How does the vote on Chair/Co-Chair work? Is it also 50%+1 or "whoever gets the most votes"?

Reply:
I will add this